### ISSUE TITLE
fix editor icons for fontawesome 5

### ISSUE THREAD

**Issue by bonesoul** [froala/wysiwyg-editor#2726]
I've been started using fontawesome 5 now and froala editor has to be fixed for it;

https://fontawesome.com/how-to-use/upgrading-from-4

**Comment by stefanneculai** [comment]
@bonesoul please make sure that you're using the editor version 2.7.6.

**Comment by bonesoul** [comment]
I'm already on 2.7.6.
The thing is that froala tries to use icons like "fa fa-smile-o" which are dropped in 5.x. Should be now far fa-smile.

**Comment by stefanneculai** [comment]
https://www.froala.com/wysiwyg-editor/examples/font-awesome-5-js :)

**Comment by bonesoul** [comment]
Your site still uses 4.4.0

`  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.4.0/css/font-awesome.min.css">`

The thing is that if you continue on using old named-icons like fa-smile-o, you'll be basically forcing all users to "SVG with JavaScript only" with the need of extra version 4 shim that converts those old icon names to 5.x compatible ones.

Basically in my project i just use webfont / css version (https://fontawesome.com/get-started/web-fonts-with-css).

So basically I'm forced to js version with extra shim now..

If you could just replace those old icon names as described here, https://fontawesome.com/how-to-use/upgrading-from-4 (Icon Name Changes Complete List) everything would settle up.

**Comment by stefanneculai** [comment]
If you don't want to use the JS way, you could include the CSS for FA5 and simply change the template option: `iconsTemplate: 'font_awesome_5s'`. We're not planning on replacing by default the Font Awesome 4.4 with 5 because the free icons which come with FA5 are not looking so good and many of our users are not looking to buy a Font Awesome license. 

A glimpse into the future, we're planning on removing the Font Awesome dependency all together and add some custom built icons.

**Comment by bonesoul** [comment]
iconsTemplate: 'font_awesome_5' didn't fix the issue for me :/.

Using fontawesome 5.0.9 and getting this;

![untitled](https://user-images.githubusercontent.com/73215/38666205-107834d8-3e47-11e8-8286-00e940dbc91d.png)

**Comment by bonesoul** [comment]
sorry, iconsTemplate change worked out.
